evalml/pipelines/components/transformers/preprocessing/time_series_featurizer.py

```diff
@@ -124,12 +124,15 @@ def fit(self, X, y=None):
         """
         if self.time_index is None:
             raise ValueError("time_index cannot be None!")
-        self.statistically_significant_lags = self._find_significant_lags(
-            y,
-            conf_level=self.conf_level,
-            start_delay=self.start_delay,
-            max_delay=self.max_delay,
-        )
+        if isinstance(y, pd.DataFrame):
+            self.statistically_significant_lags = [self.start_delay]
```

Comment:
So for the multiseries case, do we not try to find the significant lags? Does this just use all or none of the lags?

Reply:
Might want to add an explicit comment here for the case we're splitting on...e.g. if y is a dataframe, we expect it to be multiseries.